connector/github/github.go

```diff
@@ -411,12 +416,89 @@ func (c *githubConnector) user(ctx context.Context, client *http.Client) (user,
 		return u, fmt.Errorf("%s: %s", resp.Status, body)
 	}
 
-	if err := json.NewDecoder(resp.Body).Decode(&u); err != nil {
+	if err = json.NewDecoder(resp.Body).Decode(&u); err != nil {
```

Comment:
nit: using `err :=` in an if statement ensure that the variable is used only in that if statement. Did you remove the initialization for any particular reason?